nix/solidity-package.sh

```diff
@@ -17,8 +17,8 @@ mapfile -t files < <(find "$dir/src" -name '*.sol')
 json_file="out/dapp.sol.json"
 (set -x; solc $REMAPPINGS "${opts[@]}" $solcFlags "${files[@]}" > "$json_file")
 
-if [[ "$doCheck" == 1 ]] && command -v dapp2-test-hevm >/dev/null 2>&1; then
-  DAPP_OUT=out dapp2-test-hevm
+if [[ "$doCheck" == 1 ]] then
```

Comment:
I'm getting:
```
line 20: syntax error near unexpected token `then'
```

I think this should fix it:

```suggestion
if [[ "$doCheck" == 1 ]]; then
```